src/wp-includes/functions.php

```diff
@@ -8762,3 +8762,147 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
```

Comment:
@sabernhardt Thanks for handling this move in this PR!

Can we also move the test files for consistency?
- `tests/phpunit/tests/admin/wpAdminNotice.php` -> `tests/phpunit/tests/functions/wpAdminNotice.php`
- `tests/phpunit/tests/admin/wpGetAdminNotice.php` -> `tests/phpunit/tests/functions/wpGetAdminNotice.php`

Reply:
[73126ee](https://github.com/WordPress/wordpress-develop/pull/5178/commits/73126ee58e846be2017a694a1e577a4f352db6bc) Thanks!